INFO     test_api:my_log.py:48 本次执行的是测试环境用例
INFO     test_api:my_log.py:48 获取到的结果是{'code': 500, 'success': False, 'msg': '金额错误'}
INFO     test_api:my_log.py:48 需要运行的模块是['gift_dealer_manage', 'H5', 'super_admin_manage'],类型是<class 'list'>